packages/schema/src/schema/jcad.json

```diff
@@ -43,6 +43,8 @@
         "Part::MultiFuse",
         "Part::MultiCommon",
         "Part::Extrusion",
+        "Edge::Chamfer",
```

Comment:
The `Part::` prefix comes from FreeCAD, and it also uses `Part::Chamfer` and `Part::Fillet` in a FCStd file, should we keep the same schema here?